src/base/bittorrent/sessionimpl.cpp

```diff
@@ -2147,135 +2147,119 @@ void SessionImpl::processShareLimits()
 {
     qDebug("Processing share limits...");
 
+    auto resolveLimitValue = [](auto limit, auto useGlobalLimit, auto globalLimit)
```

Comment:
```suggestion
    const auto resolveLimitValue = [](const auto limit, const auto useGlobalLimit, const auto globalLimit)
```
Can't you use specific types for the parameters? (and specify a return type?)

Reply:
> Can't you use specific types for the parameters?

No, I can't, because the type is different for different limits. However, I rewrote it to explicitly indicate that the type of all parameters and the return value is the same.

Reply:
I see. It is better now. 👍